diff --git a/node_modules/import-in-the-middle/hook.mjs b/node_modules/import-in-the-middle/hook.mjs
index 7581f92..0427c1a 100644
--- a/node_modules/import-in-the-middle/hook.mjs
+++ b/node_modules/import-in-the-middle/hook.mjs
@@ -4,7 +4,7 @@
 
 const specifiers = new Map()
 
-const EXTENSION_RE = /\.(js|mjs|cjs)$/
+const EXTENSION_RE = /\.(js|mjs|cjs|ts|mts|cts)$/
 
 const NODE_MAJOR = Number(process.versions.node.split('.')[0])
 
